src/portable_python/external/xcpython.py

```diff
@@ -340,7 +340,7 @@ def auto_select_reason(self):
 
     @property
     def url(self):
-        return f"https://zlib.net/zlib-{self.version}.tar.gz"
+        return f"https://zlib.net/fossils/zlib-{self.version}.tar.gz"
```

Comment:
I've released a new version now, the real fix is to bump version of zlib to 1.3.1
The `/fossils/` end point is probably used as an archive?

I see the url is still at top level of zlib.net, they may have just retired `1.3`?
Do you happen to know that `/fossils/` is the url that should be used in general? (I don't see it mentioned on the site)

Reply:
Yes, `/fossils/` should be the right URL to use. It is on the main page of zlib, with this text: [All released versions of zlib](https://zlib.net/fossils)

Reply:
>the real fix is to bump version of zlib to 1.3.1

Without this patch, it is likely to break again when they release zlib-1.3.2.

Reply:
Thank you, yes indeed I missed that!